### PR TITLE
Create storage.md

### PR DESCRIPTION
**Pull request by cboelling**
Create component description for storage component.

### CODE REVIEW

**Review by dshorthouse — APPROVED** [review]

File: component_descriptions/storage.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Functional Component: storage
+
+## Specific scope of the module
+Collection items have storage locations. Representations of storage locations are necessary for collection management, in particular for all processing requiring information on the storage location of a given object and processes changing the location of an object, temporarily or permanently.
+
+## What does the module do? (envisioned feature set)
```
Comment (falkogloeckler) [inline component_descriptions/storage.md]: Use term `component` instead of `module`

File: component_descriptions/storage.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  - identifiers and references to the unit e.g. RFID tags, PIDs, “vernacular” names
+  - administrative details: phone numbers, people permitted, emergency contacts
+- relations between storage locations, hierarchical
+- links to object states as far as they are contingent on storage conditions
+
+Realize the referencing of storage locations for
+- representing object location
+- representing changes in object location / full history of changes
+
+## What does the module not do? (excluded features to avoid misperception / confusion)
```
Comment (falkogloeckler) [inline component_descriptions/storage.md]: `component` instead of `module`

**Review by falkogloeckler — APPROVED** [review]

File: component_descriptions/storage.md (hunk 1 above)
Comment (cboelling) [reply component_descriptions/storage.md]: OK.

File: component_descriptions/storage.md (hunk 2 above)
Comment (cboelling) [reply component_descriptions/storage.md]: OK

**Pull request merged**